lib/chunkifier.js

```diff
@@ -0,0 +1,37 @@
+const chunkifyArray = (messageLines, maxLength, delimiter) => {
```

Comment:
What does this do? Is this used somewhere or is it meant to be imported by our users?

Reply:
It's called [here](https://github.com/juliuscc/semantic-release-slack-bot/blob/c5123b64429b0ae692c0c1b26d02b849b52b478b/lib/chunkifier.js#L31).

I figured the logic deserved it's own function. For readability, extendability, and usability. :)